src/hardware/mixer.cpp

```diff
@@ -801,25 +806,31 @@ void MixerChannel::UpdateCombinedVolume()
 	combined_volume_gain = user_volume_gain * app_volume_gain * db0_volume_gain;
 }
 
-const AudioFrame MixerChannel::GetUserVolume() const
+const AudioFrame MixerChannel::GetUserVolume()
 {
+	std::lock_guard lock(mutex);
```

Comment:
Unless taking the mutex lock requires non-const, which would make sense.

Reply:
As per above.